### ISSUE TITLE
[Adaptive Cards-'Agenda.json']: At 200% resize settings, 'Redmond', '8a - 12:30p' and '12:30 - 3p' places and timings text present on the 'Agenda.json' card is getting cropped and not properly visible on the application.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9219]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0

**Pre-requisites:**
1. Go to system Settings-> System-> Display-> Scale & Layout-> Change the size of text, apps, and other items at 100%-> Display Resolution (1680*1050)
2. Go to system Settings-> Accessibility-> Text size-> 200

**Repro steps:**
1. Open the Adaptive Cards Visualizer application.
2. Navigate to the 'Agenda.json' control present on left navigation using tab key and activate it using enter key.
3. 'Agenda.json' adaptive card will appear on right sight of the pane.
4. Navigate to the 'Redmond', '8a - 12:30p' and '12:30 - 3p' places and timings text.
5. Verify that, at 200% resize settings, 'Redmond', '8a - 12:30p' and '12:30 - 3p' places and timings text present on the 'Agenda.json' card is properly visible on the application or not.

**Actual Result:**
At 200% resize settings, 'Redmond', '8a - 12:30p' and '12:30 - 3p' places and timings text present on the 'Agenda.json' card is getting cropped and not properly visible on the application.

**Expected Result:**
At 200% resize settings, 'Redmond', '8a - 12:30p' and '12:30 - 3p' places and timings text present on the 'Agenda.json' card should be properly visible without getting crop/overlap on the application.

**User Impact:**
Low vision users depend on the resize settings will be impacted if at 200% resize settings, 'Redmond', '8a - 12:30p' and '12:30 - 3p' places and timings text present on the 'Agenda.json' card is getting cropped and not properly visible on the application due to which users may be unable to know the complete information of the places and timings and might get confused on missing some informative information which lead to confusion, incorrect inputs, and a frustrating user experience.

**WCAG Reference link:**
https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/d8b35002-d101-4a40-9c6b-986da84831f8)

### Card JSON

```JSON
N/A
```
